x-pack/plugins/ml/public/settings/filter_lists/list/table.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+/*
+ * React table for displaying a table of filter lists.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+
+import chrome from 'ui/chrome';
+import { DeleteFilterListModal } from '../components/delete_filter_list_modal';
+
+
+function getColumns() {
+
+  const columns = [
+    {
+      field: 'filter_id',
+      name: 'ID',
+      render: (id) => (
+        <EuiLink href={`${chrome.getBasePath()}/app/ml#/settings/filter_lists/edit_filter_list/${id}`} >
+          {id}
+        </EuiLink>
+      ),
+      sortable: true
+    },
+    {
+      field: 'description',
+      name: 'Description',
+      sortable: true
+    },
+    {
+      field: 'item_count',
+      name: 'Item count',
+      sortable: true
+    },
+    {
+      field: 'used_by',
+      name: 'In use',
+      render: (usedBy) => {
```

Comment:
nit, rather than having the logic in here, it could be moved to a new component function. e.g. 
```
render: (usedBy = {}) => {
  <UsedBy jobs={usedBy.jobs} />
```